include/electionguard/ballot.hpp

```diff
@@ -330,6 +330,14 @@ namespace electionguard
         //TODO: void setNonce(ElementModQ *nonce);
     };
 
+    typedef enum valid_contest_return_e {
```

Comment:
The other enums start with eg_

Might want to change this one as well to be consistant.